overlord/ifacestate/udevmonitor/udevmon.go

```diff
@@ -50,13 +49,21 @@ type Monitor struct {
 	monitorStop   chan struct{}
 	netlinkErrors chan error
 	netlinkEvents chan netlink.UEvent
+
+	// devices lookup keeps track of all observed devices to know
```

Comment:
`// devices keeps track of ...`

Reply:
Done (and renamed to `seen` per other comment).